static/src/javascripts/bootstraps/enhanced/main.js

```diff
@@ -11,6 +11,7 @@ import { trackPerformance } from 'common/modules/analytics/google';
 import { init as geolocationInit } from 'lib/geolocation';
 import { init as initAcquisitionsLinkEnrichment } from 'common/modules/commercial/acquisitions-link-enrichment';
 import { fetchAndRenderEpic, fetchAndRenderHeaderLinks } from "common/modules/commercial/contributions-service";
+import { init as initCoreVitals } from 'common/modules/analytics/coreVitals';
```

Comment:
In the future we could extract this as a library and make it a bit more generic so it could be used by other Guardian sites (so long as they use Ophan), but I think it's right to just copy what we have in DCR for now.

Reply:
Yeah, I think we'll wait a little longer for version 2 to be out in the wild and then make the library switch.